inc/Classes/Translation.php

```diff
@@ -300,79 +295,64 @@ public function xml_write_db_to_file($module)
-
-                if ($row['de'] != '') {
-                    $entry .= $xml->write_tag('de', $row['de'], 4);
-                }
-
-                if ($row['en'] != '') {
-                    $entry .= $xml->write_tag('en', $row['en'], 4);
-                }
-
-                if ($row['es'] != '') {
-                    $entry .= $xml->write_tag('es', $row['es'], 4);
-                }
-
-                if ($row['fr'] != '') {
-                    $entry .= $xml->write_tag('fr', $row['fr'], 4);
-                }
-
-                if ($row['nl'] != '') {
-                    $entry .= $xml->write_tag('nl', $row['nl'], 4);
-                }
+            $entry = $xml->write_tag('id', $row['id'], 4);
+            $entry .= $xml->write_tag('org', $row['org'], 4);
 
-                if ($row['it'] != '') {
-                    $entry .= $xml->write_tag('it', $row['it'], 4);
+            $languagesToAdd = ['de', 'en', 'es', 'fr', 'nl', 'it'];
+            foreach ($languagesToAdd as $languageShort) {
+                if ($row[$languageShort] != '') {
+                    $entry .= $xml->write_tag($languageShort, $row[$languageShort], 4);
                 }
-
-                $entry .= $xml->write_tag('file', $module, 4);
-                $content .= $xml->write_master_tag('entry', $entry, 3);
             }
+
+            $entry .= $xml->write_tag('file', $module, 4);
+            $content .= $xml->write_master_tag('entry', $entry, 3);
         }
         $db->free_result($res);
 
-        // Read long Translation
+        // Read long Translations
         $res2 = $db->qry('
-            SELECT * 
+            SELECT
+                `id`,
+                `org`,
+                `de`,
+                `en`,
+                `es`,
+                `fr`,
+                `nl`,
+                `it`
             FROM %prefix%translation_long 
             WHERE 
-              file = %string% 
-              AND obsolete = 0', $module);
-        while ($row2 = $db->fetch_array($res2)) {
+              `file` = %string%
+              AND `obsolete` = 0', $module);
+        while ($row = $db->fetch_array($res2)) {
             $entry = $xml->write_tag('id', $row['id'], 4);
             $entry .= $xml->write_tag('org', $row['org'], 4);
 
-            if ($row['de'] != '') {
-                $entry .= $xml->write_tag('de', $row['de'], 4);
-            }
-
-            if ($row['en'] != '') {
-                $entry .= $xml->write_tag('en', $row['en'], 4);
-            }
-
-            if ($row['es'] != '') {
-                $entry .= $xml->write_tag('es', $row['es'], 4);
-            }
-
-            if ($row['fr'] != '') {
-                $entry .= $xml->write_tag('fr', $row['fr'], 4);
-            }
-
-            if ($row['nl'] != '') {
-                $entry .= $xml->write_tag('nl', $row['nl'], 4);
-            }
+            $languagesToAdd = ['de', 'en', 'es', 'fr', 'nl', 'it'];
```

Comment:
same here

Reply:
Fixed in https://github.com/lansuite/lansuite/pull/654/commits/d3bd13d050946bd78ce428444e760f3b7d0121fc